pkg/asset/store/store.go

```diff
@@ -357,3 +372,27 @@ func (s *storeImpl) purge(excluded []asset.WritableAsset) error {
 func increaseIndent(indent string) string {
 	return indent + "  "
 }
+
+// configDriveImage create config drive iso with ignition-config file
+// refer to https://coreos.com/os/docs/latest/config-drive.html
+func configDriveImage(file string) error {
+	logrus.Infof("Creating config drive image for %s", file)
+	if err := os.MkdirAll("openstack/latest", 0755); err != nil {
+		return err
+	}
+	cmd := exec.Command("cp", file, "openstack/latest/user_data")
+	_, err := cmd.Output()
+	if err != nil {
+		return err
+	}
+	cmd = exec.Command("mkisofs", "-R", "-V", "config-2", "-o", "configdrive.iso", ".")
```

Comment:
are you sure it's mkisofs? I think it's genisoimage?

Reply:
I think they makes the same effect.

Reply:
Maybe related:
https://github.com/weldr/lorax/blob/master/share/templates.d/99-generic/s390.tmpl#L79